package.json

```diff
@@ -9,20 +9,16 @@
   ],
   "license": "MIT",
   "peerDependencies": {
-    "@axe-core/puppeteer": "^4.4.4",
-    "axe-core": "^4.4.3"
+    "@axe-core/puppeteer": "^4.4.4"
```

Comment:
I kept this version range wider to avoid a breaking change (but it still kinda sorta is since the newer version of @axe-core/puppeteer is needed to make npm allow the newer puppeteer version, although it probably would work fine with passing `--force` to npm)